### DISCUSSION TITLE
How to submit the task?

### DISCUSSION THREAD

**Discussion by ArkaPrabhaChowdhury** [open-sauced-craftwork/browser-extension#31] [Q&A] [answered]
- Create a new branch in the private repo of the classroom 
- Push the code there

Is this right?

**Comment by nickytonline** [comment, marked as answer]
A new branch with all your code pushed to it is the way to go. And after that, you need to create a pull request so it can be reviewed. More info here if you need, https://github.com/open-sauced-craftwork/browser-extension/discussions/8#discussioncomment-5268586.

**Reply by ArkaPrabhaChowdhury** [reply]
Yes I need to create a new branch but where should this branch be? Should it be in the browser-extension repo? Or in the private GitHub classroom repo?

**Reply by ArkaPrabhaChowdhury** [reply]
@bdougie Could you please help me out here? How do I create a pull request? In the private repo or in the browser-extension repo?

**Reply by bdougie** [reply]
You create the PR in your repo. Branches in the public repo will expose your work for others to view. 

See @Anush008's comment here

https://github.com/open-sauced-craftwork/browser-extension/discussions/34#discussioncomment-5373114
